### ISSUE TITLE
FEATURE REQUEST - cpush.exe - more informative message about https://push.chocolatey.org/

### ISSUE THREAD

**Issue by bcurran3** [chocolatey/choco#1320]
Chocolatey v0.10.6 Professional
ACTION: Please update your apikey to use
'https://push.chocolatey.org/'
instead of
'https://chocolatey.org/'.
The latter source url is now considered deprecated and will not be
checked as the default source in Chocolatey v1.0. For details, run
choco apikey -?
The message would be more descriptive if it said to run "choco apikey -k -s https://push.chocolatey.org" and reference https://chocolatey.org/docs/commandsapikey for more info.

Documentation at https://chocolatey.org/docs/commandsapikey section Connecting to Chocolatey.org needs to be updated with the new source URL.

I might be missing something, but I don't see a command to delete an apikey and source URL combination.
I edited C:\ProgramData\chocolatey\config\chocolatey.config to remove the old apikey/source combo.

If desired/required to remove the old apikey/source combo, those instructions should be added to https://chocolatey.org/docs/commandsapikey as well.

I see now the api/source combo issue has been opened under:
chocolatey/choco#1301

**Comment by AdmiringWorm** [comment]
> The message would be more descriptive if it said to run "choco apikey -k -s https://push.chocolatey.org" and reference https://chocolatey.org/docs/commandsapikey for more info.

Agree, that would definitely be more descriptive, I'm thinking to that maybe leave out the first part though (in case the user sometime in the future uses an old version of choco, and the push url have changed).

> Documentation at https://chocolatey.org/docs/commandsapikey section Connecting to Chocolatey.org needs to be updated with the new source URL.

Probably just lagging a little behind (just a guess), it's after all just a couple of days since 0.10.6 was released.

> I might be missing something, but I don't see a command to delete an apikey and source URL combination.
I edited C:\ProgramData\chocolatey\config\chocolatey.config to remove the old apikey/source combo.

I was sure this was supposed to be added in 0.10.6, have I dreamed the whole thing (haven't upgraded yet, so can't say for sure), or more likely I'm just wrong and remember something else.

**Comment by ferventcoder** [comment]
> Documentation at https://chocolatey.org/docs/commandsapikey section Connecting to Chocolatey.org needs to be updated with the new source URL.

As @AdmiringWorm mentioned, this is updated already and will be pushed up to the website soon. See https://github.com/chocolatey/choco/wiki/CommandsPush and https://github.com/chocolatey/choco/wiki/CommandsApiKey

**Comment by ferventcoder** [comment]
> > I might be missing something, but I don't see a command to delete an apikey and source URL combination.
>>I edited C:\ProgramData\chocolatey\config\chocolatey.config to remove the old apikey/source combo.

>I was sure this was supposed to be added in 0.10.6, have I dreamed the whole thing (haven't upgraded yet, so can't say for sure), or more likely I'm just wrong and remember something else.

That is #1301

**Comment by ferventcoder** [comment]
Folks this is up for grabs - the source code for those pages is in this source, as you can get the same help at `choco apikey -?`.  See https://github.com/chocolatey/choco/blob/297887d526e99c0227ec60bbfb06966de24bb587/src/chocolatey/infrastructure.app/commands/ChocolateyApiKeyCommand.cs#L83-L104

**Comment by stefan-muc** [comment]
Additionally it needs a ``choco apikey -s https://chocolatey.org/ --remove`` to remove the old push url

**Comment by ferventcoder** [comment]
@stefan-muc I thought we had removed it already?

**Comment by stefan-muc** [comment]
The bug starter said "I might be missing something, but I don't see a command to delete an apikey and source URL combination.
I edited C:\ProgramData\chocolatey\config\chocolatey.config to remove the old apikey/source combo." - my entry was telling how to do it via commandline (and yes, I didn't recognize this sentence before).

Does that answer your question? If not: What "it" has already been removed?

**Comment by ferventcoder** [comment]
Ah I see, in 2017 there was not a way to remove an api key from the command line, but that has been added since. Thanks for the clarification!
